rewrite-java-test/src/test/java/org/openrewrite/java/NoStaticImportTest.java

```diff
@@ -47,4 +47,33 @@ void test() {
           )
         );
     }
+
+    @Test
+    void verifyInnerCallsAreNotUpdated() {
+        this.rewriteRun(spec -> spec.recipe(new NoStaticImport("*..* *(..)")), Assertions.java("""
+        package org.openrewrite.java;
+
+        public class TestNoStaticImport {
+
+            public static void method0() {
+            }
+
+            public static void method1() {
+                method0();
+            }
+        }
+        """, """
+        package org.openrewrite.java;
+
+        public class TestNoStaticImport{
+
+            public static void method0() {
+            }
+
+            public static void method1() {
+                method0();
+            }
+        }
```

Comment:
If you want to assert there are no code changes, then you do not have to pass the second parameter.

In this case I only saw a change in whitespace (`TestNoStaticImport {` versus `TestNoStaticImport{`) which I don't think was intentional, correct?

Reply:
```suggestion
```

Reply:
Ok, yes, correct, the before and after should be the same in that case.